src/main/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogTableGeneratorSpanner.java

```diff
@@ -64,9 +87,18 @@ public Sql[] generateSql(
       Database database,
       SqlGeneratorChain sqlGeneratorChain) {
     String databaseChangeLogTableName = getDatabaseChangeLogTableNameFrom(database);
-    String createTableSql =
-        this.createTableSql.replaceAll("__DATABASECHANGELOG__", databaseChangeLogTableName);
-    return new Sql[] {new UnparsedSql(createTableSql)};
+
+    Dialect dialect = null;
+    try {
+      dialect = ((ICloudSpanner) database).getDialect();
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
nit (we can fix this in a follow-up pull request): Can we remove the `throws SQLException` declaration from `getDialect()`, and instead add this try-catch-throw-RuntimeException block to the `getDialect()` method? Otherwise, I'm afraid that we'll just repeat this block in a large number of places that needs to know the dialect.